modules/op-query-results-name.adoc

```diff
@@ -0,0 +1,81 @@
+
+.Procedure
+
+. To list the names of all results that correspond to pipeline runs and task runs created in a namespace, run the following command:
++
+[source, terminal]
+----
+$ opc results list --addr ${RESULTS_API} <namespace-name>
+----
++
+.Example command and output
+[source, terminal]
+----
+$ opc results list --addr ${RESULTS_API} results-testing
+Name                                                          Start                                   Update
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed  2023-06-29 02:49:53 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/ad7eb937-90cc-4510-8380-defe51ad793f  2023-06-29 02:49:38 +0530 IST           2023-06-29 02:50:06 +0530 IST
+results-testing/results/d064ce6e-d851-4b4e-8db4-7605a23671e4  2023-06-29 02:49:45 +0530 IST           2023-06-29 02:49:56 +0530 IST
+----
+
+. To list the names of all records in a result, run the following command:
++
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} <result-name>
+----
++
+.Example command and output
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed
+Name                                                                                                       Type                                    Start                                   Update
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e9c736db-5665-441f-922f-7c1d65c9d621  tekton.dev/v1beta1.TaskRun              2023-06-29 02:49:53 +0530 IST           2023-06-29 02:49:57 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/5de23a76-a12b-3a72-8a6a-4f15a3110a3e  results.tekton.dev/v1alpha2.Log         2023-06-29 02:49:57 +0530 IST           2023-06-29 02:49:57 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/57ce92f9-9bf8-3a0a-aefb-dc20c3e2862d  results.tekton.dev/v1alpha2.Log         2023-06-29 02:50:05 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e9a0c21a-f826-42ab-a9d7-a03bcefed4fd  tekton.dev/v1beta1.TaskRun              2023-06-29 02:49:57 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/04e2fbf2-8653-405f-bc42-a262bcf02bed  tekton.dev/v1beta1.PipelineRun          2023-06-29 02:49:53 +0530 IST           2023-06-29 02:50:05 +0530 IST
+results-testing/results/04e2fbf2-8653-405f-bc42-a262bcf02bed/records/e6eea2f9-ec80-388c-9982-74a018a548e4  results.tekton.dev/v1alpha2.Log         2023-06-29 02:50:05 +0530 IST           2023-06-29 02:50:05 +0530 IST
+----
+
+. To retrieve the YAML code for a pipeline run or task run from a record, run the following command:
++
+[source, terminal]
+----
+opc results records get --addr ${RESULTS_API} <record-name> | yq -r .data.value | base64 -d | yq -P '.'
```

Comment:
I tried it now and `opc` returns json, not yaml so `yq` doesn't work

Reply:
@khrm please advise as to the correct command to retrieve the YAML manifest from a record, I got this one from http://docs.openshift-pipelines.org/results/query-using-opc.html

Reply:
Sorry, I did not notice it earlier. The possible output formats are tab, textproto and JSON. The default is tab in tkn-results. I am not sure about opc. However, you can use `jq` instead of `yq`.

Reply:
yaml is not even an option for output
```
opc results records get [flags] <record_path>

  <record name>: Fully qualified name of the record. This is typically "<namespace>/results/<result name>/records/<record uid>".


Flags:
  -h, --help            help for get
  -o, --output string   output format. Valid values: textproto|json (default "json")
  ```

Reply:
It's working fine for me. YQ works for `json`, `xml` and `yaml`.

Reply:
@ppitonak @khrm  this needs to be resolved today or the PR might not make GA date

Reply:
OK, the issue was indeed in different implementations and version of `yq`. I verified that this one works fine https://github.com/mikefarah/yq/releases/tag/v4.35.1

Reply:
By `yq` here, we mean https://github.com/mikefarah/yq 👼🏼 Other `yq` tool might not work with json…

Reply:
Also, the output would be in JSON form instead of yaml form.

Reply:
But we can simulate what's yq doing with this:
```

./opc results records get --insecure  --addr ${RESULTS_API} default/results/01b5b0f2-0280-4a88-a836-4dbeed6452d7/records/01b5b0f2-0280-4a88-a836-4dbeed6452d7   | jq -r .data.value | base64 -d |xargs -0 python3 -c 'import sys, yaml, json; j=json.loads(sys.argv[1]); print(yaml.safe_dump(j))'
```